packages/suite/src/views/settings/coins/index.tsx

```diff
@@ -1,88 +1,36 @@
-    }));
-
-    const unavailableCapabilities = device?.unavailableCapabilities ?? {};
-
-    const mainnetNetworksFilterFn = (n: Network) => !n.accountType && !n.testnet;
-
-    const testnetNetworksFilterFn = (n: Network) => {
-        if (n.symbol === 'regtest' && !debug.showDebugMenu) {
-            return false;
-        }
-        return !n.accountType && 'testnet' in n && n.testnet === true;
-    };
+const StyledSettingsLayout = styled(SettingsLayout)`
+    & > * + * {
+        margin-top: 16px;
+    }
+`;
 
-    const unavailableNetworksFilterFn = (symbol: Network['symbol']) =>
-        !unavailableCapabilities[symbol];
-
-    const enabledMainnetNetworks: Network['symbol'][] = [];
-    const enabledTestnetNetworks: Network['symbol'][] = [];
-
-    enabledNetworks.forEach(symbol => {
-        const network = NETWORKS.find(n => n.symbol === symbol);
-        if (!network) return;
-        if (network.testnet) {
-            enabledTestnetNetworks.push(network.symbol);
-        } else {
-            enabledMainnetNetworks.push(network.symbol);
-        }
-    });
+const Settings = () => {
+    const { mainnets, testnets, enabledNetworks, setEnabled } = useEnabledNetworks();
 
     return (
-        <SettingsLayout>
-            <CoinsGroup
-                label={<Translation id="TR_COINS" />}
-                description={<Translation id="TR_COINS_SETTINGS_ALSO_DEFINES" />}
-                enabledNetworks={enabledMainnetNetworks}
-                filterFn={mainnetNetworksFilterFn}
-                onToggleOneFn={changeCoinVisibility}
-                onActivateAll={() =>
-                    changeNetworks([
-                        ...enabledTestnetNetworks.filter(unavailableNetworksFilterFn),
-                        ...NETWORKS.filter(mainnetNetworksFilterFn)
-                            .map(n => n.symbol)
-                            .filter(unavailableNetworksFilterFn),
-                    ])
-                }
-                onDeactivateAll={() => changeNetworks(enabledTestnetNetworks)}
-                type="mainnet"
-            />
-
-            <CoinsGroup
-                label={<Translation id="TR_TESTNET_COINS" />}
-                description={<Translation id="TR_TESTNET_COINS_EXPLAINED" />}
-                enabledNetworks={enabledTestnetNetworks}
-                filterFn={testnetNetworksFilterFn}
-                onToggleOneFn={changeCoinVisibility}
-                onActivateAll={() =>
-                    changeNetworks([
-                        ...enabledMainnetNetworks.filter(unavailableNetworksFilterFn),
-                        ...NETWORKS.filter(testnetNetworksFilterFn)
-                            .map(n => n.symbol)
-                            .filter(unavailableNetworksFilterFn),
-                    ])
-                }
-                onDeactivateAll={() => changeNetworks(enabledMainnetNetworks)}
-                type="testnet"
-            />
-        </SettingsLayout>
+        <StyledSettingsLayout>
```

Comment:
add titles to be consistent with other sections

Reply:
https://github.com/trezor/trezor-suite/pull/4779/commits/cdbebe64bb56d46e7387e506b12375aab8b31f74